packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -104,7 +121,10 @@ export default class extends Component {
   async componentDidMount() {
     const plugin = await getPlugin('netdata')
     const isNetDataPluginCorrectlySet = plugin !== undefined && plugin.loaded
-    this.setState({ isNetDataPluginCorrectlySet })
+    this.setState({
+      isNetDataPluginCorrectlySet,
+      schedGran: await getSchedulerGranularity(this.props.host),
```

Comment:
If the granularity change, the UI will not be updated. Better subscribe to its change.